### PR TITLE
Trainee Subject Index

### PR DESCRIPTION
**Pull request by shakil-fg**

### CODE REVIEW

File: app/views/user_subjects/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<h1><%= t :user_subject %></h1>
+<div class="row">
+  <div class="col-sm-7 col-sm-offset-0">
+    <% if !(@subjects.empty?) %>
+      <% @subjects.each do |subject| %>
+        <div class="jumbotron">
+          <%= link_to user_subject_path(subject) do %>
```
Comment (mdmahmudurrahman) [inline app/views/user_subjects/index.html.erb]: Please remove()

File: app/views/user_subjects/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<h1><%= t :user_subject %></h1>
+<div class="row">
+  <div class="col-sm-7 col-sm-offset-0">
+    <% if !(@subjects.empty?) %>
```
Comment (mdmahmudurrahman) [inline app/views/user_subjects/index.html.erb]: Please remove()

**Comment by noor-framgia** [comment]
Ok

**Comment by mdmahmudurrahman** [comment]
Okey

**Comment by manazirahsan** [comment]
okey

**Pull request merged**